src/screens/ImportSeedPhraseSheet.js

```diff
@@ -361,16 +351,11 @@ export default function ImportSeedPhraseSheet() {
 
   useEffect(() => {
     if (isImporting) {
-      setComponent(
-        <LoadingOverlay
-          paddingTop={keyboardVerticalOffset}
-          title={walletLoadingStates.IMPORTING_WALLET}
-        />,
-        true
-      );
-      return hide;
+      setIsWalletLoading(walletLoadingStates.IMPORTING_WALLET);
```

Comment:
minor: can we replace the if else with something like `setIsWalletLoading(isImporting ? walletLoadingStates.IMPORTING_WALLET : null);`